lib/beacon/component_instances.ex

```diff
@@ -0,0 +1,75 @@
+defmodule Beacon.ComponentInstances do
```

Comment:
Hey @cibernox can you please describe the purpose of `ComponentInstance`? I'm not following why this is necessary, I may be missing something.

Reply:
Since I wanted to leave the `Beacon.component` table alone, I created this one. From a ComponentDefinition, which is like a blueprint, we create ComponentInstances which is the ones we assign to a specific page.

Do you reckon I should use the `Beacon.Content.Component` type instead of created a new one?